utils/lists.ts

```diff
@@ -85,6 +85,11 @@ export const resourceList: Record<string, string>[] = [
     description: "Graphing community projects for adventurers to explore",
     url: "https://loot-bibliotheca-client.vercel.app/",
   },
+  {
+    name: "Loot info",
+    description: "View what derivatives have been claimed by a Loot Bag",
+    url: "https://loot-info-client.herokuapp.com/projects/loot"
```

Comment:
right now when I click the URL, it would redirect to me to "https://loot-info-client.herokuapp.com/".

could you please fix the redirect (I recommend that)? otherwise you might just leave "https://loot-info-client.herokuapp.com" as it makes no difference.

Reply:
good catch - will fix @zhew2013

Reply:
fixed @zhew2013

Reply:
Small nit: could you add a comma in the end? Just following the convention of this repo